src/compute_bond_local.cpp

```diff
@@ -86,6 +88,8 @@ void ComputeBondLocal::init()
 {
   if (force->bond == NULL)
     error->all(FLERR,"No bond style is defined for compute bond/local");
+  if (comm->ghost_velocity == 0)
```

Comment:
@efremdan1 This test should only be made, if the user requested a property requiring ghost velocities. For just potential energy or force, ghost velocities are not needed.